xds/internal/httpfilter/rbac/rbac.go

```diff
@@ -144,6 +152,12 @@ func (builder) BuildServerInterceptor(cfg httpfilter.FilterConfig, override http
 	}
 
 	icfg := c.config
+	// "If absent, no enforcing RBAC policy will be applied" - RBAC
+	// Documentation for Rules field.
+	if icfg.Rules == nil {
+		return &interceptor{}, nil
```

Comment:
This can just return `nil, nil` I believe.
https://github.com/grpc/grpc-go/blob/master/xds/internal/httpfilter/httpfilter.go#L75

Reply:
Yeah good point, switched.